registry.yaml

```diff
@@ -2800,6 +2800,11 @@ packages:
   files:
   - name: aws-nuke
     src: 'aws-nuke-{{.Version}}-{{.OS}}-{{.Arch}}'
+- type: github_release
+  repo_owner: replicatedhq
+  repo_name: kots
+  asset: 'kots_{{.OS}}_{{.Arch}}.tar.gz'
+  description: KOTS provides the framework, tools and integrations that enable the delivery and management of 3rd-party Kubernetes applications, a.k.a. Kubernetes Off-The-Shelf (KOTS) Software
```

Comment:
```suggestion
  description: KOTS provides the framework, tools and integrations that enable the delivery and management of 3rd-party Kubernetes applications, a.k.a. Kubernetes Off-The-Shelf (KOTS) Software
  files:
  - name: kots
  - name: kubectl-kots
    src: kots
```

I'm not familiar with this tool, but this tool seems to be kubectl plugin.

https://kots.io/kots-cli/getting-started/

Reply:
Thanks. You're right!